Sources/HaCWebsiteLib/Models/Link.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+/// Models a hyperlink
+struct Link {
+  let title: String
```

Comment:
If we're mapping this to HTML `a` elements, will `title` become its contents or the `title` attribute? [MDN on `title`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title)

Reply:
I was thinking contents. Think title is a reasonable name for it. Would `text` be clearer? `linkText`?

Reply:
yeah, I think text is clearer